### PR TITLE
Added druid exporter in the list

### PR DESCRIPTION
**Pull request by iamabhishek-dubey**
Signed-off-by: Abhishek Dubey abhishekbhardwaj510@gmail.com

### CODE REVIEW

**Comment by brian-brazil** [comment]
Thanks for your PR. We try to avoid duplication in this list, what does this do that the JMX exporter can't?

**Comment by roidelapluie** [comment]
By looking at the README I see druid_segement_* metrics. Is that a typo? segement vs segment.

**Comment by iamabhishek-dubey** [comment]
Hey @roidelapluie 
Thanks for the comment, yes it was a typo and I have fixed :)

**Comment by iamabhishek-dubey** [comment]
Hey @brian-brazil 
Thanks for the comment, yes initially we tried JMX exporter with druid but we were not getting the supervisors, task metrics. That's why we created this

**Comment by brian-brazil** [comment]
https://druid.apache.org/docs/latest/operations/metrics.html seems to have task metrics

**Comment by iamabhishek-dubey** [comment]
Yes but supervisor metrics are not there so shall I modify exporter for supervisors metrics only?

**Comment by brian-brazil** [comment]
It's more understanding if this is a new thing that deserves a listing, or if it's only mildly different. I have a sense that this is more akin to kube-state-metrics, but the docs of both this exporter and druid itself aren't very clear.

**Comment by iamabhishek-dubey** [comment]
Okay @brian-brazil I am closing the PR but thanks for the feedbacks

**Comment by iamabhishek-dubey** [comment]
One last question, can we list this under miscellaneous section like kube-state-metrics or do you think this needs more maturity?

**Comment by brian-brazil** [comment]
It's not a question of maturity, it's a question of if this is doing things that are fundamentally different enough from what is possible with the jmx exporter.

**Comment by iamabhishek-dubey** [comment]
Understood, so I will again compare the metrics between these two and then reopen the PR if needed until then I am closing the PR. Thanks for feedbacks

**Comment by iamabhishek-dubey** [comment]
I have tested the druid with JMX exporter but it only gives the JVM data, we are not actually getting the metrics of the druid. Here are the example metrics:-
https://gist.github.com/iamabhishek-dubey/5ef19d3db9deb25475a80c9ff5c79262

Again re-opening the PR :-)

**Comment by iamabhishek-dubey** [comment]
Also, I have updated the documentation for better clarity.

**Comment by brian-brazil** [comment]
That's still not very clear, and it makes it seem as though druid itself doesn't expose anything via JMX.

The key question for me is is this monitoring druid itself, or the data inside druid? In general an exporter should only be doing one of these things.

**Comment by iamabhishek-dubey** [comment]
Yes, druid doesn't expose its metrics via JMX, and this exporter is to monitor druid and its services not the data inside it.

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [SQL exporter](https://github.com/free/sql_exporter)
    * [Tarantool metric library](https://github.com/tarantool/prometheus)
    * [Twemproxy](https://github.com/stuartnelson3/twemproxy_exporter)
+   * [Druid Exporter](https://github.com/opstree/druid-exporter)
```
Comment (brian-brazil) [inline content/docs/instrumenting/exporters.md:57]: Keep it sorted.

Comment (iamabhishek-dubey) [reply content/docs/instrumenting/exporters.md]: Done

**Comment by iamabhishek-dubey** [comment]
I have made the suggested changes

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**